specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/1.2-preview/iotcentral.json

```diff
@@ -0,0 +1,5643 @@
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      },
+      "put": {
+        "operationId": "FileUploads_Create",
+        "summary": "Create the file upload storage account configuration.",
+        "x-ms-examples": {
+          "Create file upload storage account configuration": {
+            "$ref": "./examples/fileUpload_update.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/Version"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "File upload storage account configuration body.",
+            "schema": {
+              "$ref": "#/definitions/FileUpload"
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/FileUpload"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      },
+      "patch": {
+        "operationId": "FileUploads_Update",
+        "summary": "Update the file upload storage account configuration",
+        "x-ms-examples": {
+          "Update file upload storage account configuration": {
+            "$ref": "./examples/fileUpload_update.json"
+          }
+        },
+        "consumes": [
+          "application/json",
+          "application/json-patch+json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/Version"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "File upload storage account configuration body.",
+            "schema": {
+              "type": "object"
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/FileUpload"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      },
+      "delete": {
+        "operationId": "FileUploads_Remove",
+        "summary": "Delete the file upload storage configuration.",
+        "x-ms-examples": {
+          "Remove file upload storage configuration": {
+            "$ref": "./examples/fileUpload_delete.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/Version"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Success"
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
```

Comment:
The path for a put should have a final path parameter.

Reply:
File upload is settings instead of real file uploads you can specify an id, we are considering to move this settings to application in future, there is no id returned or specified.